### ISSUE TITLE
`--sshd-image`/`--rsync-image` not working

### ISSUE THREAD

**Issue by Frankkkkk** [utkuozdemir/pv-migrate#137]
**Describe the bug**
Hi,

On `pv-migrate` built from commit `2a19f03363820bd89be9f3aba6bc1c5248b02283`, the options `--sshd-image`/`--rsync-image` are not taken into account and the default image is always used.

**To Reproduce**
Launch pv migrate with different images:`pv-migrate migrate XXX --strategies lbsvc --rsync-image img-test:2 --sshd-image test-img:1 pvc1 pvc2`

Describe the new launched pod:
```
Containers:
  rsync:
    Container ID:  
    Image:         docker.io/utkuozdemir/pv-migrate-rsync:1.0.0
---
  Normal  Pulling                 31s   kubelet                  Pulling image "docker.io/utkuozdemir/pv-migrate-rsync:1.0.0"
```

**Expected behavior**
The images specified in the argument should be used instead

**Console output**
n/a

**Version**
 - Source and destination Kubernetes versions 1.21
 - `pv-migrate` version and architecture see above
 - 
Thanks and cheers !

**Comment by utkuozdemir** [comment]
Many thanks, seems I introduced tons of regressions lately :) Will look into it.

I also need to to improve the tests to take these things into consideration.

**Comment by Frankkkkk** [comment]
Haha, no problem, that's why there are integration testers 😉 😛 
Have a nice week !

**Comment by utkuozdemir** [comment]
After some thinking, I decided to remove the image and serviceAccount related flags. They can already be passed as Helm values, so the flags were redundant.

I added an example to customize the image into the README: https://github.com/utkuozdemir/pv-migrate#examples

The flags are removed from the just-released version `v0.8.0` on.

Please let me know if it works for you or if you have any other issues.
